CautionaryAlertsApi/V1/Gateways/UhGateway.cs

```diff
@@ -131,6 +133,26 @@ public async Task<IEnumerable<CautionaryAlertListItem>> GetCautionaryAlertsByMMH
             return alerts.Select(x => x.ToDomain());
         }
 
+        public CautionaryAlert GetCautionaryAlertByAlertId(AlertQueryObject query)
+        {
+            var alerts = _uhContext.PropertyAlertsNew
+                        .Where(x => x.MMHID == query.PersonId.ToString())
+                        .Where(x => x.AlertId == query.AlertId.ToString())
+                        .ToList();
```

Comment:
You should consider doing the ToList() at the very end.
ToList() brings all the data from the db to the client (the lambda)
Because you can run the count() operations before a a ToList(), the operation is run in memory without retrieving the data. And that is much more efficient.